### DISCUSSION TITLE
Please also watch!

### DISCUSSION THREAD

**Discussion by xiaoshu123i** [tabler/tabler#1339] [General]
Why does China not include Taiwan on the home page of the Tabler?

**Comment by xiaoshu123i** [comment]
为什么在Tabler的首页地图上面台湾是不属于中国的？

**Reply by codecalm** [reply]
that is a question to contributors of [`jsvectormap`](https://github.com/themustafaomar/jsvectormap), we just use it library in Tabler :)
